### DISCUSSION TITLE
Tasmota for new Platforms - ESPHome moved to libretiny

### DISCUSSION THREAD

**Discussion by he-so** [arendst/Tasmota#21201] [Ideas]
Hi,
for some time, I recognise the trend that iot devices shipped with integrated esp chips are more and more hard to find on ecommerce platforms. In consequence, tasmotizable devices, without soldering or transplant required, are getting rare. The tasmota device database is becoming more and more a heap of dead links pointing to products that are no more on stock. Even though, if you find i.e. a shutter switch in the database that can still be phurchased, chances are high that it no more inclues esp chips, but it was silently replaced by some beken alternative.
What happens outside the tasmota world? Meanwhile there are open source firmware alternatives for those pin compatible esp12 boards like the tuya wb3s. To mention some; openbeken, that ships with a confusig frontend. But, also ESPHome was successfully ported to to bk7231 world.

For me, as a long time tasmota user, I'd like to stay with tasmota. So I am wondering if it might be consired, porting tasmota to bk7231 aswell. As of now, I did not find any strategical or technical discussion of tasmotas future. But my guess is, tasmota will have a hard time to survive, when it only runs on brain transplanted devices.

Feedback welcome.

regards,
heso

**Comment by Jason2866** [comment]
Nope, porting is not considered. It would need a total rewrite of Tasmota to support this platform.
There a no resources for. Theo and the few people of the inner development circle just don't have the resources for.
If no one is using anymore Tasmota RiP. It is not a commercial product.

**Comment by openshwprojects** [comment]
[OpenBeken](https://github.com/openshwprojects/OpenBK7231T_App) has basic compatibility layer with, for example, [Tasmota JSON format](https://www.youtube.com/watch?v=OhdkEJ-SuTU), we have also a [teardowns list with over 500 devices supported](https://openbekeniot.github.io/webapp/devicesList.html), so you can give it a try when you encounter BK7231T, BK7231N, BK7231M, BL2028N, or even W600, W800 (W601/W801), LN882H, XR809, BL602 and more...
